### ISSUE TITLE
[Feature request] Auto streak

### ISSUE THREAD

**Issue by MohamadBayz** [rodit/SnapMod#223]
Add an option for Auto streak that the app sends a specific image that we select to a specific friends in a specific time in the day daily as for the streak if it's possible to send a random image from a folder of our image.

**Comment by Lucky-069** [comment]
Let alone this. There has been no update for over a month. I assume maintaining the mod itself is very time consuming since snapchat keeps on pushing new base with changes in almost every base. I think just maintaining the mod is best one can do.

**Comment by MohamadBayz** [comment]
Yeah, it's great what he's doing takes a lot of nerve and time to do so.
Well, we don't need casual updates for supporting more version probably once in a while update for fixing bugs and adding new features would be good and couple months once an update for supporting a newer version too
